bee-message/src/output/feature_block/dust_deposit_return.rs

```diff
@@ -28,6 +33,8 @@ impl TryFrom<u64> for DustDepositReturnFeatureBlock {
 impl DustDepositReturnFeatureBlock {
     /// The [`FeatureBlock`](crate::output::FeatureBlock) kind of a [`DustDepositReturnFeatureBlock`].
     pub const KIND: u8 = 2;
+    /// Valid amounts for a [`DustDepositReturnFeatureBlock`].
+    pub const AMOUNT_RANGE: RangeInclusive<u64> = DUST_DEPOSIT_MIN..=IOTA_SUPPLY;
```

Comment:
This will be gone soon anyway, but would an exclusive range make more sense? You wouldn't have enough IOTAs for a value transaction anymore if everything is deposited as dust protection.

Reply:
I will let it like that for now and discuss it with the RFC and Hornet guys, it needs to be the exact same bound in both hornet and bee otherwise one may discard the message and not the other and one of them won't be able to solidify

Reply:
Ah, forget about that, I'm being dumb. It will be replaced by the byte cost 🤦

Reply:
If you don't mind I'll just leave it like that until we remove